src/CardsAPI/Hand.py

```diff
@@ -1,13 +1,11 @@
-from .Card import Card
+from src.CardsAPI.Card import Card
 from typing import List, Tuple
 
 
 class Hand(object):
-    def __init__(self, card_list: List[Card]=None, is_dealer_hand: bool=False,
-                 isSplit: bool = False):
-        if card_list is None:
-            card_list = []
-        self.card_list = card_list
+    def __init__(self, card_list: List[Card] = None, is_dealer_hand:
+                 bool = False, isSplit: bool = False):
+        self.card_list = card_list or []
```

Comment:
It's not easier to initialise this default value in the `__init__` ?

Reply:
Nope, mutable default arguments are evil.
https://docs.python-guide.org/writing/gotchas/

Reply:
Ok, my python vision is so _"Bisounours"_, thanks for the link !